frontend/src/pages/notebookController/StartServerModal.tsx

```diff
@@ -34,27 +60,22 @@ const StartServerModal: React.FC<StartServerModalProps> = ({
   );
 
   const running = () => (
-    <>
-      <p className="odh-notebook-controller__start-server-modal-text">
-        Server ready at {notebook?.metadata.annotations?.['opendatahub.io/link']}
-      </p>
-      <ActionList>
-        <ActionListItem>
-          <Button
-            onClick={() => window.open(notebook?.metadata.annotations?.['opendatahub.io/link'])}
-          >
-            Access server
-          </Button>
-        </ActionListItem>
-        <ActionListItem>
-          <Button variant="link" onClick={() => setStartModalShown(false)}>
-            Notebook server control panel
-          </Button>
-        </ActionListItem>
-      </ActionList>
-    </>
+    <p className="odh-notebook-controller__start-server-modal-text">
+      Server ready at {notebook?.metadata.annotations?.['opendatahub.io/link']}
```

Comment:
Do we not want this to be a clickable link? It looks rough in its current state

Reply:
It will be confusing to the user if the link is clickable because the page will be redirected soon. The UX here may just want the user to wait until redirecting.

Reply:
Hmmm -- yeah you're right. But I feel we should stylize it in some fashion... it looks really uncomfortable to have a long URL string that is not in anyway stylized. Let us push this off for now, we can get feedback from UX after it is up and running.